### ISSUE TITLE
<img> tags not checked with using a closing slash

### ISSUE THREAD

**Issue by petdance** [petdance/html-lint#15]
1. <img src='test.gif' /> # gives me no errors
2. <img src='test.gif'>
   # gives 2 errors as expected
   # <IMG SRC="test.gif"> does not have ALT text defined
   # <IMG SRC="test.gif"> tag has no HEIGHT and WIDTH attributes

Comment 1 by 4x4owner, Oct 28, 2010

  i just had same issue [amongst others..]

```
  # create a Lint object, sets strict_names = 1 by default.
  #see - http://search.cpan.org/~gaas/HTML-Parser-3.68/Parser.pm 
  $lint = HTML::Lint->new();

  # overwrite strict_names. allows parse of incomplete tags eg.. <img ... />
  # [may also generate some unwanted output...]
  $lint->{_parser}->strict_names(0);
```
